pymedphys/tests/dicom/test_anonymise.py

```diff
@@ -169,6 +169,27 @@ def test_anonymised_dataset_with_empty_patient_sex():
     assert hardcode_replace_ds["PatientSex"].value is None
 
 
+@pytest.mark.slow
+@pytest.mark.pydicom
+def test_alternative_identifying_keywords():
+    default_keyword_list = list(IDENTIFYING_KEYWORDS)
```

Comment:
Is this line needed in light of the following line and it's review comment?

Reply:
No.  Once it's clear that one is making a copy to start with.
I took one approach, you have identified an alternative approach that expresses the semantics intended in a clearer fashion.  I'm happy to switch to something that reads more clearly.